pages/documentation/developer/repositories/powsybl-single-line-diagram_guide1.md

```diff
@@ -0,0 +1,114 @@
+# Generating my first single line diagrams - Guide 1
```

Comment:
Add the layout:
```
---
layout: default
---

Reply:
I think the title is not ok, we should give contextual information `Single Line Diagram - Generating my first diagram`?

Reply:
As it is a (single) API guide at the end, I chose a less tutorial-ish title: "Single Line Diagram - SVG Writing"